### PR TITLE
#34: Add .editorconfig that defines 4 char ident for all files, but p…

### PR DESCRIPTION
**Pull request by iocanel**
…om.xml where ident uses tabs.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
```
Comment (spencergibb) [inline .editorconfig]: All Spring Projects use tabs

Comment (iocanel) [inline .editorconfig]: Even better! I'll fix that asap! :-)

**Comment by iocanel** [comment]
@spencergibb: does it look ok now?

**Comment by spencergibb** [comment]
Looks good now. We could probably add rules that are currently defined in https://github.com/spring-cloud/spring-cloud-build/blob/master/spring-cloud-dependencies-parent/eclipse-code-formatter.xml as an enhancement.

**Pull request merged**